apps/wasm/mvp/src/client.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::*;
+use proof_systems::mina_signer::create_kimchi;
+use proof_systems::mina_signer::{NetworkId, PubKey, Signer};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(typescript_custom_section)]
+const CLIENT_OPTIONS: &'static str = r#"
```

Comment:
I don't think we need to export all these js APIs, are 2 functions of signing and sending sufficient for this?

Reply:
yes I will be updating this to export only singing and sending func